includes/notifications.php

```diff
@@ -262,3 +276,42 @@ function pmpro_footer_link() {
 	<?php }
 }
 add_action( 'wp_footer', 'pmpro_footer_link' );
+
+/**
+ * Check if a plugin is active with a specific version.
+ * @param array $checks Plugin data to run a check. Includes: [ 0 => plugin-slug, 1 => comparison operator, 2 => version_to_check]
+ */
+function pmpro_check_plugin_version( $checks = NULL ) {
+	// Assume pass is failing right now.
+	$pass = false;
+
+	// run check if plugin data is not emptpy.
+	if ( $checks ) {
```

Comment:
Even if you're checking that there's something in $checks, the code may still trigger PHP notices since there are no checks for whether $checks[0] and $checks[1] actually exist.

Reply:
FYI https://github.com/strangerstudios/paid-memberships-pro/commit/f82f458bee963c5367f6a03a982ae7bbe5279e6a

Reply:
Going to make a lot more updates today yet